app/src/org/commcare/tasks/UpdateTask.java

```diff
@@ -236,6 +238,10 @@ public void setAsAutoUpdate() {
         wasTriggeredByAutoUpdate = true;
     }
 
+    public void setAsLocalUpdate() {
+        isLocalUpdate = true;
+    }
```

Comment:
Hmm, and instead of the `isLocalUpdate` field, you can have `authority` be a class field